cryptosecretbox/crypto_secretbox_easy.go

```diff
@@ -48,13 +48,15 @@ func CryptoSecretBoxEasy(m []byte, n []byte, k []byte) ([]byte, int) {
 		(C.ulonglong)(len(m)),
 		(*C.uchar)(&n[0]),
 		(*C.uchar)(&k[0])))
-
-	return c, exit
+	nonceAndEncrypted := support.BytesCombine(n, c)
```

Comment:
This seems a change in functionality: a developer can combine the nonce and ciphertext if that's required.

Also: why doesn't `append(n, c...)` suffice?